jssha/jssha.d.ts

```diff
@@ -1,13 +1,22 @@
 // Type definitions for jsSHA
 // Project: https://github.com/Caligatio/jsSHA
-// Definitions by: David Li <https://github.com/randombk>
-// Definitions: https://github.com/borisyankov/DefinitelyTyped
```

Comment:
should this line be 
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
?